README.md

```diff
@@ -1,264 +1,505 @@
+
+```bash
+npm start
+```
 
-Run Tests:
+Run all linters:
 
 ```bash
-npm run test # or `npm test`
+npm run lint
 ```
 
-Create a new component folder in Pattern Lab with scss, twig, md, & yml/json by running:
+Run all tests:
 
 ```bash
-npm run new
+npm test
 ```
 
-To update node and composer dependencies (if you see messages about Pattern Lab wanting to Merge or Replace files, merge them):
+Run accessibility testing on Pattern Lab rendered output:
 
 ```bash
-npm run update
+npm run test:accessibility
 ```
 
-## Assets
+To update node and composer dependencies (**merge** if offered the option):
 
-Get front end libraries downloaded with Bower like so:
+```bash
+npm run update
+```
 
-    bower install {project-name} --save
+## Structure
+
+The following are significant items at the root level:
+
+    # ./
+    .
+    ├── apps                           # Things that use the compiled design system. Drupal theme & PL
+    ├── dist                           # Bundled output: CSS, js, images, app artifacts (like PL html)
+    ├── source                         # The design system. All assets compiled to dist/
+    ├── tools                          # Gulp plugins and node tools
+    ├── gulpfile.js                    # Defines the few tasks required in the workflow
+    ├── webpack.drupal.config.js       # Entry point for the Drupal theme bundle
+    ├── webpack.pl.config.js           # Entry point for the Pattern Lab bundle
+    ├── webpack.shared.config.js       # Shared bundle configuration for all entry points
+    └── ...                            # Mostly just config
+
+`source/` holds all assets for the design system and looks like this:
+
+    # ./source/
+    .
+    ├── _patterns                      # All assets live within an Atomic "pattern"
+    │   ├── 01-atoms                   # Twig namespace: @atoms, JS/Sass namespace: atoms
+    │   │   ├── button                 # For instance, the button atom
+    │   │   │    ├── demo              # Patterns feature a demo folder to show implementation
+    │   │   │    │   ├── buttons.twig  # Demonstrate with a plural name, visible to PL since no underscore
+    │   │   │    │   └── buttons.yml   # Data provided to the demo pattern
+    │   │   │    ├── _button.scss      # Most components require styles, underscore required
+    │   │   │    ├── _button.twig      # The pure component template, underscore required
+    │   │   │    └── index.js          # Component entry point (See "Anatomy of a Component below)
+    │   │   └── ...                    # Other @atoms
+    │   └── ...                        # @base, @atoms, @molecules, @organisms, @templates, @pages
+    └── design-system.js               # The ultimate importer/exporter of the design system pieces
+
+>The design system is *consumed by* "apps". The two apps included are a Drupal theme and a Pattern Lab installation.
+
+`app/pl/` holds the *entry point* for all Pattern Lab assets, as well as the PHP engine:
```

Comment:
What does entry point mean in this context?

Reply:
Entry point is defined within this document. Should we link to that section of the document at this text?